Signal/src/call/UserInterface/CallUIAdapter.swift

```diff
@@ -47,7 +47,7 @@ extension CallUIAdaptee {
         AssertIsOnMainThread()
 
         guard self.callService.call == nil else {
-            assertionFailure("unexpectedly found an existing call when trying to call back: \(recipientId)")
+            Logger.debug("unexpectedly found an existing call when trying to start outgoing call: \(recipientId)")
```

Comment:
can you bump it up to `info` so that we get it in production builds? 

Seems like a source of potentially confusing behavior in the wild, and is not likely to spam the logs a bunch.